code/__HELPERS/logging.dm

```diff
@@ -118,15 +118,26 @@
  * @param amount The amount of units transferred
  * @param reagent_names List of reagent names to log
  */
-/proc/log_reagents(var/mob/user, var/source, var/target, var/amount, var/list/reagent_names)
-	if (amount == 0)
+/proc/log_reagent_transfer(var/mob/user, var/atom/source, var/atom/target, var/amount_desired)
```

Comment:
It would require passing user too and looked really fucking ugly when I did it
Think of it as a helper proc